src/app/services/project_service.js

```diff
@@ -138,18 +138,30 @@ angular
             _.each(service.files.manifest.sources, function(node) {
                 node.label = "" + node.source_name + "." + node.name;
                 service.files.manifest.nodes[node.unique_id] = node;
+                node.color = '#5fb825'
             });
 
             // Add exposures back into nodes to make site logic work
             _.each(service.files.manifest.exposures, function(node) {
                 node.label = node.name;
                 service.files.manifest.nodes[node.unique_id] = node;
+                node.color = '#ff694b'
             });
             
             // Add metrics back into nodes to make site logic work
             _.each(service.files.manifest.metrics, function(node) {
                 node.label = node.name;
                 service.files.manifest.nodes[node.unique_id] = node;
+                node.color = '#ff5688'
+            });
+
+            _.each(service.files.manifest.nodes, function(node) {
+                if (node.docs && node.docs.color) {
```

Comment:
again - this logic is kind of jank, but it works! We should definitely clean this up if we intend to merge code like this. Big idea:
- use the `docs -> color` property if one is specified
- otherwise, use a default node color

I am not sure if I'm incorporating the `node.color` attributes set just above this.. might need to workshop that part :)